stable/nginx-ingress/values.yaml

```diff
@@ -36,6 +36,10 @@ controller:
   ##
   kind: Deployment
 
+  ## enable RBAC as per: https://github.com/kubernetes/ingress/tree/master/examples/rbac/nginx and https://github.com/kubernetes/ingress/issues/266
+  rbac:
+    enabled: false
```

Comment:
fyi, looks like the Traefik chart is merged w/ `rbac.enabled`, :+1:

https://github.com/kubernetes/charts/blob/master/stable/traefik/values.yaml#L63